wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -177,10 +181,10 @@ public final static class Files {
         public static final String QRINFO_BOOTSTRAP_FILENAME = "qrinfo" + FILENAME_NETWORK_SUFFIX + ".dat";
 
         /** Filename of the fees files. */
-        public static final String FEES_FILENAME = "fees" + FILENAME_NETWORK_SUFFIX + ".txt";
+        public static final String FEES_FILENAME = "fees" + FEE_NETWORK_SUFFIX + ".txt";
 
         /** Filename of the file containing Electrum servers. */
-        public static final String ELECTRUM_SERVERS_FILENAME = "electrum-servers.txt";
+        public static final String ELECTRUM_SERVERS_FILENAME = "electrum-servers" + FILENAME_NETWORK_SUFFIX + ".txt";
```

Comment:
sweeping was primarily done using an electrum server rather than a block explorer.  This change will allow electrum servers to be supported on Testnet.

Reply:
Development on the electrum servers stopped in March of this year.  Perhaps it is better to have these as a backup to the block explorers instead.

Devnets don't have electrum servers running, so the block explorer is used instead.